package/MDAnalysis/units.py

```diff
@@ -342,6 +343,14 @@
         assert not unit in unit_types  # see comment!
         unit_types[unit] = utype
 
+#: Lookup table for base units in MDAnalysis by unit type.
```

Comment:
Very nice.

Could you please also add to the caption of the base unit table https://github.com/MDAnalysis/mdanalysis/pull/3749/files#diff-7fde25a4fd35872588976243972caeaebea3711006ad4283815eb3f68be1fd2eR39 a reference to 
```reST
.. Table:: Base units in MDAnalysis as encoded in :data:`MDANALYSIS_BASE_UNITS`.
```